package/rtcm3_out_protocol/src/rtcm3_out_router.yml

```diff
@@ -0,0 +1,18 @@
+
+name: RTCM3_ROUTER
```

Comment:
Do we need a separate router for this, can we integrate it with the existing rtcm router?  It seems like this config and the existing one are opening server sockets on the some addresses.

Reply:
Actually, I don't think this file is used since there doesn't appear to be a new endpoint router launched with this config.

Reply:
yea this file should be removed

Reply:
removed